src/app/ReadClient.h

```diff
@@ -345,6 +357,18 @@ class ReadClient : public Messaging::ExchangeDelegate
     InteractionModelEngine * mpImEngine = nullptr;
     ReadPrepareParams mReadPrepareParams;
     uint32_t mNumRetries = 0;
+
+    // Reserved size for the FabricFiltered boolean flag, which takes up 1 byte for the control tag and 1 byte for the context tag.
+    static constexpr uint16_t kFabricFiltered = 1 + 1;
+    // End Of Container (0x18) uses one byte.
+    static constexpr uint16_t kReservedSizeForEndOfContainer = 1;
+    // Reserved size for the uint8_t InteractionModelRevision flag, which takes up 1 byte for the control tag and 1 byte for the
+    // context tag, 1 byte for value
+    static constexpr uint16_t kReservedSizeForIMRevision = 1 + 1 + 1;
```

Comment:
Where are we using kReservedSizeForIMRevision?

Reply:
Fixed, thanks